src/pipeline.rs

```diff
@@ -709,6 +713,31 @@ mod tests {
         // non-transactional did not
         assert!(onramp2_rx.is_empty());
 
+        // send a cb insight
+        let event_id = EventId::from((0, 0, 1));
+        addr.send_insight(Event {
+            id: event_id.clone(),
+            cb: CbAction::Close,
+            ..Event::default()
+        })
+        .await?;
+
+        // transactional onramp received it
+        // chose exceptionally big timeout, which we will only hit in the bad case that stuff isnt working, normally this should return fine
+        match timeout(Duration::from_millis(10000), onramp_rx.recv()).await {
```

Comment:
would it make sense to const that timeout?

Reply:
dont know, but i did it :)

Reply:
:+1: you always say that so I wanted to say it once too :joy:

Reply:
that is only fair 😆